azure-devops/golang/stages/40-delivery/arm.yaml

```diff
@@ -203,7 +187,43 @@ stages:
                     --rule-name "$(PIPELINE_FIREWALL_NAME)" \
                     --start-ip-address "$PIPELINE_IP"
             displayName: 'Create Flexible Firewall Rule to execute migrations and seeders'
-            condition: and(succeeded(), or(eq(variables['MIGRATIONS_CACHE_HIT'], 'false'), eq(variables['SEEDERS_CACHE_HIT'], 'false')))
+
+          - script: |
+              set -e
+
+              PORT=$(echo $OUTPUTS | jq -r '.databaseServerPort.value')
```

Comment:
Can you please create a template for that? Is being very repetitive the same thing in this file. Look at the line 238 and 265.

Reply:
I create another step to create the variable, please review that